pkg/api/types.go

```diff
@@ -962,10 +962,13 @@ type ContainerStatus struct {
 	Ready bool `json:"ready"`
 	// Note that this is calculated from dead containers.  But those containers are subject to
 	// garbage collection.  This value will get capped at 5 by GC.
-	RestartCount int    `json:"restartCount"`
-	Image        string `json:"image"`
-	ImageID      string `json:"imageID"`
-	ContainerID  string `json:"containerID,omitempty"`
+	RestartCount      int              `json:"restartCount"`
+	ContainerID       string           `json:"containerID,omitempty"`
+	Image             string           `json:"image"`
+	ImageID           string           `json:"imageID"`
+	ImageCreationTime unversioned.Time `json:"imageCreationTime,omitempty"`
+	ImageSize         int64            `json:"imageSize,omitempty"`
```

Comment:
I assume this value will match the actual size of the container. I don't think we should expose the details of image management through the APIs. For all we know, we might start serializing all images to help with image accounting on the nodes.